src/components/visualizations/chart/highcharts/commonConfiguration.ts

```diff
@@ -1,9 +1,10 @@
-// (C) 2007-2018 GoodData Corporation
+// (C) 2007-2019 GoodData Corporation
 import cloneDeep = require("lodash/cloneDeep");
 import invoke = require("lodash/invoke");
 import get = require("lodash/get");
 import set = require("lodash/set");
 import isEmpty = require("lodash/isEmpty");
+import { css } from "highcharts";
```

Comment:
hmm just wondering it we could create some sort of highcharts API wrapper module which will be providing usage of highcharts API :) what do you think?

Reply:
yes. i would like to consider to improve later.